fms/models/gpt_bigcode.py

```diff
@@ -13,8 +13,12 @@
 
 @dataclass
 class GPTBigCodeConfig(ModelConfig):
-    src_vocab_size: int = 49157  # This param default is based on https://huggingface.co/bigcode/gpt_bigcode-santacoder
-    emb_dim: int = 2048  # This param default is based on https://huggingface.co/bigcode/gpt_bigcode-santacoder
+    src_vocab_size: int = (
+        49157  # This param default is based on https://huggingface.co/bigcode/gpt_bigcode-santacoder
```

Comment:
hmm... this file hasn't been changed in weeks. what do we think caused lint suddenly to fail? maybe a version change somewhere in the version of the linter that's installed by CI? we might just want to pin the version.

If we can't just fix the version used in CI and are to change the code,  for this particular line, it looks like the issue is with the lenght of the line. Rather than using this kind of awkward parenthesis change I think I'd rather just move the comment to the line above the assignment.